standalone/src/main/java/io/pravega/local/InProcPravegaCluster.java

```diff
@@ -278,6 +278,8 @@ private void startLocalSegmentStore(int segmentStoreId) throws Exception {
                         .with(ServiceConfig.KEY_FILE, this.keyFile)
                         .with(ServiceConfig.CERT_FILE, this.certFile)
                         .with(ServiceConfig.ENABLE_TLS_RELOAD, this.enableTlsReload)
+                        .with(ServiceConfig.LISTENING_IP_ADDRESS, this.enableTls ? "localhost" : "")
+                        .with(ServiceConfig.PUBLISHED_IP_ADDRESS, this.enableTls ? "localhost" : "")
```

Comment:
@Tristan1900 one more ask: could you please extract the `localhost` string into a constant and use that here and above, so that it can be easily changed in one place (to say loopback address `127.0.0.1` or another interface during testing).

Reply:
Also, curious isn't the condition used here wrong? It sets `LISTENING_IP_ADDRESS` and `PUBLISHED_IP_ADDRESS` to empty string, if TLS is not enabled. Am I missing something?

Reply:
Sure, added. Thanks for pointing out.

Empty string is the default value for those two parameters. The constructor will replace the empty string with the right ip address in [here](https://github.com/pravega/pravega/blob/0701daa7213eb8617603d2df725700c5c1ad51bb/segmentstore/server/src/main/java/io/pravega/segmentstore/server/store/ServiceConfig.java#L321)